specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -16,6 +16,7 @@
         "environmentId": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/rg/providers/Microsoft.App/managedEnvironments/demokube",
         "workloadProfileName": "My-GP-01",
         "configuration": {
+          "targetLabel": "production",
```

Comment:
i think we have a separate example file for labels and keep default as is

Reply:
i agree, fixed